test/lib/mockResponse.spec.js

```diff
@@ -1221,6 +1221,30 @@ describe('mockResponse', function () {
 
       it('should inherit from Node OutogingMessage.end()');
 
+      it("triggers callback provided as 1st argument", function() {
+        var buff = Buffer.from('random-text');
+        var called = false;
+        function cb() { called = true; }
+        response.end(cb);
+        expect(called).to.eql(true);
```

Comment:
Please use `sinon.spy()` to test that function was executed.

Here is an example https://github.com/howardabrams/node-mocks-http/blob/master/test/lib/mockResponse.spec.js#L807

Reply:
sure :)